spec/features/adventure_spec.rb

```diff
@@ -1,30 +1,180 @@
 require 'rails_helper'
 
 describe "Adventures" do
-  let!(:adventure) { FactoryBot.create(:adventure) }
+  let!(:user)               { create(:user) }
+  let!(:adventure)          { create(:adventure, user: user) }
+  let!(:adventure_password) { create(:adventure, user: user, slug: "test-story-password") }
+
+  context "when logging in/creating an account" do
```

Comment:
nomenclature comment: i'd say this block fits more of a `describe` block than a `context`. Functionally they are equivalent, but you can use the two to convey the structure of your test. `describe` = what you are testing. `context` = the situation in which your tests are occuring